src/containers/Home/reducer.js

```diff
@@ -50,6 +50,20 @@ export const reducer = (state, action) => {
         productFilters: filteredProducts,
       };
 
+    case "REQUEST_SEARCH_FILTER":
+      console.log(action.payload);
```

Comment:
`console.log` can be removed

Reply:
Yup, nice catch. Those were there for testing, will be removed.

Reply:
Removed these in other PR.